<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>881</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_check_commodity">
     <widget class="QTableWidget" name="check_commodity_view">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>751</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;所有商品&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="vacant"/>
    <widget class="QWidget" name="page_search_commodity">
     <widget class="QTableWidget" name="search_commodity_view">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>121</y>
        <width>711</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>711</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="commodity_name">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;请输入商品名称：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_off_shelf">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>711</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="commodity_ID">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;请输入商品ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="off_shelf_view">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>121</y>
        <width>711</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="check_order">
     <widget class="QTableWidget" name="check_order_view">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>751</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;所有订单&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_check_user">
     <widget class="QTableWidget" name="check_user_view">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>751</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;所有用户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_ban_user">
     <widget class="QTableWidget" name="ban_user_view">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>121</y>
        <width>711</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>711</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="user_ID">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;请输入用户ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>账户</string>
    </property>
    <addaction name="cancellation"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>商品</string>
    </property>
    <addaction name="actioncheck_commodity"/>
    <addaction name="actionsearch"/>
    <addaction name="actionoff_shelf"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>订单</string>
    </property>
    <addaction name="actioncheck_order"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actioncheck_user"/>
    <addaction name="actionban_user"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="cancellation">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actioncheck_commodity">
   <property name="text">
    <string>查看商品</string>
   </property>
  </action>
  <action name="actionsearch">
   <property name="text">
    <string>搜索商品</string>
   </property>
  </action>
  <action name="actionoff_shelf">
   <property name="text">
    <string>下架商品</string>
   </property>
  </action>
  <action name="actioncheck_order">
   <property name="text">
    <string>查看订单</string>
   </property>
  </action>
  <action name="actioncheck_user">
   <property name="text">
    <string>查看用户</string>
   </property>
  </action>
  <action name="actionban_user">
   <property name="text">
    <string>封禁用户</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
